test/arbitrum/HimalayaCompoundArbitrum.t.sol

```diff
@@ -246,6 +263,7 @@ contract HimalayaCompoundArbitrumUnitTests is HimalayaCompoundUtils, ConnextUtil
     }
     //case 2: no amounts to migrate
     else if (collateralAmount == 0 && debtAmount == 0) {
+      // TODO: this case is never reached because vm.assumes `collateralAmount` > 1e14
```

Comment:
this specific case is never reached.
I suggest to consider it.
This comment applies to all three test files:
- arbitrum, mainnet and polygon.